servant/src/Servant/Types/ResponseList.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE DeriveTraversable #-}
+
+-- | This module offers other servant libraries a minimalistic HTTP response type.
+--
+-- It is purely an internal API and SHOULD NOT be used by end-users of Servant.
```

Comment:
Should the module name be `Servant.Types.Internal.ResponseList` instead? I believe that's what we do elsewhere.

Reply:
Done